src/DomainGuest.sol

```diff
@@ -78,9 +78,9 @@ abstract contract DomainGuest {
     uint256 public lid;         // Local ordering id
     uint256 public rid;         // Remote ordering id
     uint256 public grain;       // Keep track of the pre-minted DAI in the remote escrow [WAD]
-    uint256 public sin;         // A running total of how much was already requested to parent domain to re-capitalize this domain [WAD]
+    uint256 public sin;         // Amount already requested to parent domain to re-capitalize this one but hasn't yet been paid [WAD]
```

Comment:
What about naming this quantity something different, like `ail` ? :)
Would avoid having to use `sin`, `_sin` and `sin_`.

<img width="710" alt="image" src="https://user-images.githubusercontent.com/1734245/206721541-cebadafc-232c-401c-8d85-445035546a19.png">

Reply:
Hmmm well `sin` is a pretty much known term in the system, not only in the `vat`, but also in the `vow`, and we are also using it in the `DomainHost`. So I'd keep the storage variable as it is. Maybe we should do better in the functions to avoid confusion. I'm still not sure what would be the best way.

Reply:
Yes, it's just that `domainGuest.sin()` represents the part of `vat.sin()` that is currently in transit from the guest to the host, so it sounds like it deserves a different name than `sin`. Maybe `dsin` (aka delta-sin) to emphasize that it refers to the amount by which `vat.sin()` will ultimately be reduced (after healing)? Or we could also keep it `sin` but just have the local variable `sin_` be replaced by `dsin`.

Reply:
Hmmm `dsin` doesn't sound bad as the storage variable, but I would use the same one in `DomainHost` as well.